frontend/src/components/common/Table/Table.tsx

```diff
@@ -366,6 +367,17 @@ export default function Table<RowItem extends Record<string, any>>({
     },
   });
 
+  // toggle
+  useHotkeys(
+    'ctrl+shift+t',
```

Comment:
This will collide with the browser's same shortcut for re-opening the last closed tab. Maybe `ctrl+alt+f`?
We are planning on having a way to edit the shortcuts but we don't have it yet.

Reply:
Also a shortcut overview?